burn-import/onnx-tests/tests/onnx_tests.rs

```diff
@@ -641,4 +646,69 @@ mod tests {
         let expected = Data::from([[[[1.0000, 0.5000, 0.3333, 0.2500]]]]);
         output.to_data().assert_approx_eq(&expected, 4);
     }
+
+    #[test]
+    fn cos() {
+        let model: cos::Model<Backend> = cos::Model::new();
+
+        let input = Tensor::<Backend, 4>::from_floats([[[[1.0, 4.0, 9.0, 25.0]]]]);
+
+        let output = model.forward(input);
+        let expected = Data::from([[[[0.5403, -0.6536, -0.9111, 0.9912]]]]);
+
+        output.to_data().assert_approx_eq(&expected, 4);
```

Comment:
Not related to this PR: Is it possible to move `assert_approx_eq` into `burn-common` and define our own approximation comparator there? In this way we can use that in every `burn` crate and change its implementation in a single point.

Reply:
For now it's a method used with the `Data` struct, do you have an idea how you want to generalise the method?